custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -33,20 +33,17 @@ def create_cases(self):
 
     def create_person_case(self):
         person_structure = self.person()
-        person_case = self.factory.create_or_update_case(person_structure)[0]
-        person_structure.case_id = person_case.case_id
+        self.factory.create_or_update_case(person_structure)
```

Comment:
I think you can just remove this call and the one for occurrence too. Both the person and occurrence should get created with this `self.factory.create_or_update_case(episode_structure)`.